gdpr/utils.py

```diff
@@ -0,0 +1,16 @@
+from users.models import User
+
+
+def get_user(user: User) -> User:
```

Comment:
So is this function just overriding a  helusers method with same name? I don't see this used anywhere in this PR.

Reply:
This is set in a setting `GDPR_API_USER_PROVIDER=(str, ""),`, perhaps it would be good to put this function in there as the default value, so it would be more clear.
It is explained a bit here: https://github.com/City-of-Helsinki/helsinki-profile-gdpr-api?tab=readme-ov-file#obtaining-a-user-model-instance
That package expects the gdpr model to be something that refers to the user model with the `user` attribute. However in our case I have defined this gdpr model to be the user model. This seems to be the only way to make that happen. It is quite confusing.